src/components/implementation/fork_tests/micro_fork/micro_fork.c

```diff
@@ -7,25 +7,37 @@
 #define ITER (1024)
 u64_t meas[ITER];
 
+/*
+ * Warning - test assumes Composite can run at least ITER components.
+ * (And this may not take into account other components for this test
+ */
+
 void cos_init(void)
 {
 	u64_t start, end, avg, tot = 0, dev = 0;
 	int i, j;
 	spdid_t f;
 	int iter = ITER;
 
-	call();			/* get stack */
+	call();			/* get stack */ // how does this get the stack?
```

Comment:
1. style guide
2. I'm not sure what the utility of this comment is.  If it stays in the code, it will be confusing for future people looking at the code.  If you remove it, then you likely still have question.  If you're trying to remind yourself to ask a question, then add these questions to your research log, and make sure to ask them.